src/task2.py

```diff
@@ -0,0 +1,14 @@
+from math import inf
+KOEFITIENTS = {'bosh':556, 'Руці-Лапки': 1}
+
+class DishWasher:
+    def __init__(self, mark) -> None:
```

Comment:
use empty line as to split one method end from next one

Reply:
Fixed